### ISSUE TITLE
Created campaign and it won't take

### ISSUE THREAD

**Issue by lorelei1** [Giveth/giveth-dapp#556]
There's already an [issue for this](https://github.com/Giveth/giveth-dapp/issues/436) but I wasn't sure if my comment would notify anybody since it's closed. 

I created this DAC like 2 days ago and it still looks like this...
<img width="1230" alt="screen shot 2018-12-07 at 5 45 46 pm" src="https://user-images.githubusercontent.com/38163206/49640536-cf75e700-fa48-11e8-9201-b0d6b1f7479e.png">

**Comment by vojtechsimetka** [comment]
Seems like it is fine, no? https://beta.giveth.io/dacs/5c086caed663227e436371c5

**Comment by lorelei1** [comment]
Yeah it looks great now did you do something? If not, then maybe I should make another DAC as a test, because if it was just time that did the trick, and it took 3 days to finish pending...

**Comment by vojtechsimetka** [comment]
The Giveth node was down so I am pretty sure that was the problem. We actually can’t really cancle DACs now so maybe wait with such test :).
